src/crate/theme/rtd/crate/navbar.html

```diff
@@ -15,9 +15,9 @@
             <a href="https://crate.io/products/">Products</a>
             <ul class="sub-menu">
               <li class="menu-item"><a href="https://crate.io/products/cratedb/">CrateDB</a></li>
+              <li class="menu-item"><a href="https://crate.io/products/cratedb-cloud-on-azure/">CrateDB Cloud on Azure</a></li>
               <li class="menu-item"><a href="https://crate.io/products/cratedb-cloud/">CrateDB Cloud</a></li>
               <li class="menu-item"><a href="https://crate.io/products/crate-iot-data-platform/">Crate IoT Data Platform</a></li>
-              <li class="menu-item"><a href="https://crate.io/cratedb-comparison/">CrateDB in Comparison</a></li>
```

Comment:
how come we're removing this link?

Reply:
Because we don't need it in the product overview atm